test/fields/collections/Lexical/e2e/main/e2e.spec.ts

```diff
@@ -685,8 +685,12 @@ describe('lexicalMain', () => {
     })
   })
 
-  test('creating a link, then clicking in the link drawer, then saving the link, should preserve cursor position and not move cursor to beginning of richtext field', async () => {
+  /**
+   * A fix was implemented to resolve a limitation in Firefox where the escape key resets the active element to the beginning of the page
+   */
+  test('ensure escape key can be used to move focus away from editor', async () => {
```

Comment:
Yes please.

> To avoid accidental toggles, I suggest displaying toasts with the messages:
> 
> > You have disabled the option to indent with the tab key. To enable it press escape again.
> 
> > You have enabled the option to indent with the tab key. To disable it press escape again.

My concern as I said there is accidental toggles. If I accidentally pressed escape, it would be confusing to try to indent something and not be able to. A toast would be ideal to indicate what is happening.

Reply:
It's not implemented as a permanent toggle. Whenever they re-focus the editor the tab indent plugin will automatically be enabled.

The escape key is being used in the same way as it would be used in a modal (escaping current context).

A toast just wouldn't make sense with the above in mind.

Reply:
oh I see what you mean. I figured the focus would stay on the editor and just the indent feature would be disabled.
I'm fine if the behavior is like non-Firefox browsers (focus is removed), so we can skip the toast as far as I'm concerned.